<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UICFTNClientControls</class>
 <widget class="QWidget" name="UICFTNClientControls">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDP tracker settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../facetracknoir/images/facetracknoir.png</normaloff>../facetracknoir/images/facetracknoir.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinPortNumber">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Add to axis</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>yaw</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="add_yaw">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-180</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>pitch</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>roll</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="add_pitch">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-180</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="add_roll">
        <item>
         <property name="text">
          <string>0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>+180</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>-180</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>startEngineClicked()</slot>
  <slot>stopEngineClicked()</slot>
  <slot>cameraSettingsClicked()</slot>
 </slots>
</ui>
